starter-code/src/functions-and-arrays.js

```diff
@@ -48,6 +142,24 @@ var wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray(array){
```

Comment:
this function will return 2 errors in the jasmine test. A simpler way to go about this function is to mainly compare arrays. You can do something like this 
```
function uniquifyArray(arrayOfWords){
if (arrayOfWords.length === 0){
  return undefined;
}
let resultArray= [];
 arrayOfWords.forEach((oneWord)=>{
  if (resultArray.indexOf(oneWord) === -1){
    resultArray.push(oneWord);
  }
});
return resultArray;
}
```